Mathlib/Data/Finset/Pi.lean

```diff
@@ -88,6 +88,9 @@ theorem pi_empty {t : ∀ a : α, Finset (β a)} : pi (∅ : Finset α) t = sing
   rfl
 #align finset.pi_empty Finset.pi_empty
 
+lemma pi_nonempty (ht : ∀ a ∈ s, (t a).Nonempty) : (s.pi t).Nonempty := by
```

Comment:
Should it be an `iff` lemma? May be, even `simp`?

Reply:
For some reason I convinced myself the converse didn't hold. Fixed